pkg/metrics/metrics.go

```diff
@@ -131,6 +134,26 @@ var (
 	// LabelKind is the kind a label
 	LabelKind = "kind"
 
+	// LabelPath is the label for the API path
+	LabelPath = "api_path"
```

Comment:
the `api_` prefix is not necessary. The label is only valid within the specified metric and the context is clear.